Blish HUD/GameServices/ArcDps/V2/CommonFields.cs

```diff
@@ -18,9 +18,9 @@ public class CommonFields {
         ///     Contains every player in the current group or squad.
         ///     Key: Character Name, Value: Account Name
         /// </summary>
-        public IReadOnlyDictionary<string, Player> PlayersInSquad => _playersInSquad;
+        public IReadOnlyDictionary<ulong, Player> PlayersInSquad => _playersInSquad;
```

Comment:
Can I get context on the type change?  The `summary` above still suggests that the key is a Character Name.

Reply:
The type change was necessary due to ArcDPS not sending the character name anymore when an agent gets removed, only the Agent ID. I updated the summary to reflect that change.